### ISSUE TITLE
Brave payments add funds function not working correctly

### ISSUE THREAD

**Issue by mbretttx** [brave/browser-laptop#9270]
- Did you search for similar issues before submitting this one?
Yes
- Describe the issue you encountered: At 02:58 on 6/5/17, I attempted to fund my account via Coinbase in the amount of $4.75. They sold me 0.00301664 BTC for $7.82 plus $0.99 fee for a total of $8.81.  They sent  0.00301664 BTC (7.77) to Bitcoin address 3EmQkaBgCGxGJ8fTFTPvdooUMAXXEn1p1f. However, the Brave payment screen only credited $4.75 to my account vs. the $7.77 sent by Coinbase. Not good! After this payment is disbursed, I will turn off Brave payments until this problem is resolved 

- Platform (Win7, 8, 10? macOS? Linux distro?): Win7

- Brave Version (revision SHA):Brave	0.15.314
rev	75ffa36
Muon	3.0.202
libchromiumcontent	58.0.3029.110
V8	5.8.283.38
Node.js	7.9.0
Update Channel	dev
os.platform	win32
os.release	6.1.7601
os.arch	x64

- Steps to reproduce:
    1. Add funds from Brave Payments screen (Will not attempt to reproduced as this would involve $ expenditure on my part).
    2. Verify purchase amount in Coinbase.
    3. Verify amount sent to Brave in Coinbase
    4. Compare amount sent by Coinbase to amount posted on Brave Payments screen0
    
- Actual result:

- Expected result:

- Will the steps above reproduce in a fresh profile? If not what other info can be added?

- Is this an issue in the currently released version? Yes

- Can this issue be consistently reproduced? Unknown

- Extra QA steps:
    1.
    2.
    3.

- Screenshot if needed:

- Any related issues:

**Comment by smoofra** [comment]
I'm seeing the same behavior.  not good indeed.

**Comment by mrose17** [comment]
i'm trying to understand the flow here. https://blockchain.info/address/3EmQkaBgCGxGJ8fTFTPvdooUMAXXEn1p1f shows the activity for that address (thanks for including that, by the way, it saves a round-trip on the issue).

the last transaction shown is https://blockchain.info/tx/08d2741934bfdea048e1ef8bcc2642c7a628f23f6228bb1fd59d88e1d09f2fcf which shows about the same time (modulo TZ) as your report. this shows a total transfer of $4.94 (as of the time of the transaction).

there appears to be something going on with the coinbase buy widget. i am investigating... thanks for reporting this and thanks for your patience! (cc: @alexwykoff )

**Comment by mrose17** [comment]
all - thanks for your patience as i investigated this.

tl;dr - 

- https://support.coinbase.com/customer/portal/articles/2109597-buy-sell-bank-transfer-fees
- https://support.coinbase.com/customer/portal/articles/815435-does-coinbase-pay-bitcoin-miner-fees-

@mbretttx - i have sent some coin to your address to make up for the "messaging" on the buy widget.sorry!

@smoofra - if this happened to you as well, please go to your `ledger-state.json` file, look for a line that starts with `"address":`and email the line to mrose@brave.com and i'll look into it.

__and if either of you know other folks who have been bitten by this, please ask them to do the same__ 

sorry for the situation and thanks for your patience as i attempt a resolution. (i have some ideas on a fix...)

**Comment by mrose17** [comment]
follow-up to above: for some reason, @mbretttx, the transaction didn't go through. i just checked the logs and saw that... sorry. here is a transaction that has is being confirmed now (i hope): https://blockchain.info/tx/b3e1e9e7a3695e1b5179b49b9032f24f0c8fec77be56f966d1e31643a0c0771b

**Comment by luixxiul** [comment]
closing as resolved.
